app/models/current_user.rb

```diff
@@ -30,6 +30,10 @@ def moderator?
     !!@payload['moderator']
   end
 
+  def admin?
+    !!@payload['admin']
```

Comment:
@castrolem we have an admin field in user, and this class delegates calls to the user model, see `alias __getobj__ user`. This should work without defining this method.